server/lib/queries.js

```diff
@@ -1160,17 +1160,18 @@ const getTransactionsTimeSeries = async (
 const getTagFrequencies = async args => {
   let searchTermFragment = '';
   let term = args.searchTerm;
-  // Cleanup term
+
   if (term && term.length > 0) {
+    // Cleanup term
     term = sanitizeSearchTermForILike(trimSearchTerm(term));
     if (term[0] === '@') {
       // When the search starts with a `@`, we search by slug only
       term = term.replace(/^@+/, '');
       searchTermFragment = `AND slug ILIKE '%' || :term || '%' `;
     } else {
-      searchTermFragment = `
-        AND ("searchTsVector" @@ plainto_tsquery('english', :vectorizedTerm)
-        OR "searchTsVector" @@ plainto_tsquery('simple', :vectorizedTerm))`;
+      searchTermFragment += `
```

Comment:
:ice_cream: IMO this `getTagFrequencies` has its place in `server/lib/search.js`. I'm open to other places, but `server/lib/queries.js` is a black hole that we should avoid when possible. Putting all our SQL queries in the same file seems a bit messy.

Regarding the change on the line, going from `=` to `+=` seems unnecessary and forces you to look at the previous code to understand what we're concatening our string with. Unless there's a good reason to change,I would keep it consistent and remove the `+`

Reply:
Makes sense. I've made the changes.